docker-compose.yml

```diff
@@ -78,6 +80,25 @@ services:
       - agile-core:agile
     restart: always
 
+
+
+  agile-idm:
+    image: agileiot/agile-idm-armv7l
```

Comment:
I must say: the automation to build images and upload them to dockerhub was done by @cskiraly  in the agile-idm repo, so in this regard I am clueless.

I guess versionist along with the .travis file can help with this right? If there is a project-wide way to integrate this I would not have an issue with placing this in the security-relevant repos.